### ISSUE TITLE
[Configuration] Add more global configuration

### ISSUE THREAD

**Issue by yordis** [algolia/algoliasearch-rails#85]
Configurations like:
- `raise_on_failure` which is very handy for development for every model. 
- `enqueue` probably use for every model
- `synchronous` probably use for every model
- `per_environment` probably use it all the time
- `force_utf8_encoding` probably use it for every model
- `disable_indexing` probably always disabled if I am in test environment

are not posible to configure throw `config` (`production.rb` `test.rb` `development.rb`)

**Comment by redox** [comment]
:+1:

**Comment by manzhikov** [comment]
+1

**Comment by Spone** [comment]
Any progress on this please? :disappointed:

**Comment by Spone** [comment]
Yeah, happy that the Rails integration is finally getting some :two_hearts:

**Comment by duhaime** [comment]
@julienbourdeau has there been any movement on this issue? We would very much like to configure `enqueue` callbacks in a global config
